assignment4/disjointset.py

```diff
@@ -0,0 +1,37 @@
+class DisjointSet:
+    """Implementation of a Disjoint Set. Takes integers as keys."""
+
+    def __init__(self):
+        self.parent = list()
+        self.rank = list()
+        self.setAmount = 0
+
+    def getSetAmount(self):
+        return self.setAmount
+
+    def makeSet(self, v):
+        if len(self.parent) <= v:
+            toAdd = v - len(self.parent) + 1
+            self.parent += [0] * toAdd
```

Comment:
I'm probably missing something, but doesn't toAdd only gets a value different than 1 on the first iteration? if so, why not just have the parent list initiated as a list with one cell of 0?